hoomd/hpmc/update.py

```diff
@@ -838,16 +841,21 @@ class QuickCompress(Updater):
     distributed between ``max(min_scale, 1.0 - min_move_size / max_diameter)``
     and 1.0 where ``min_move_size`` is the smallest MC translational move size
     adjusted by the acceptance ratio and ``max_diameter`` is the circumsphere
-    diameter of the largest particle type.
+    diameter of the largest particle type. If `allow_unsafe_resize` is `True`,
+    box move sizes will be uniformly distributed between ``min_scale`` and 1.0
+    (with no consideration of ``min_move_size``).
 
     Tip:
         Use the `hoomd.hpmc.tune.MoveSize` in conjunction with
         `QuickCompress` to adjust the move sizes to maintain a constant
         acceptance ratio as the density of the system increases.
 
     Warning:
-        When the smallest MC translational move size is 0, `QuickCompress`
-        will scale the box by 1.0 and not progress toward the target box.
+        When the smallest MC translational move size is 0, `allow_unsafe_resize`
+        must be set to `True` to progress toward the target box. A decrease in
+        `max_overlaps_per_particle` is recommended when using this setting. In
+        all other cases, using the default `allow_unsafe_resize` = `False` is
+        preferred, as it proposes better box moves.
```

Comment:
```suggestion
        When the smallest MC translational move size is 0, `allow_unsafe_resize`
        must be set to `True` to progress toward the target box. Decrease
        `max_overlaps_per_particle` when using this setting to prevent
        unresolvable overlaps.
```
Users decide what they prefer.